stan/math/prim/mat/fun/distance.hpp

```diff
@@ -1,6 +1,7 @@
 #ifndef STAN_MATH_PRIM_MAT_FUN_DISTANCE_HPP
 #define STAN_MATH_PRIM_MAT_FUN_DISTANCE_HPP
 
+#include <stan/math/prim/scal/fun/distance.hpp>
```

Comment:
remove this include; it's not used anywhere.

Reply:
removed